programmers/문자열-내-p와-y의-개수-세기.js

```diff
@@ -0,0 +1,33 @@
+function solution(s){
+    var count_p = 0;
```

Comment:
```suggestion
    let count_p = 0;
```
ES6에 대해서 한번 공부해보시는 것을 추천드려요  자바스크립트에서는 변수를 선언하는 방식이 세가지가 있는데 
사용하신 var,  let,  const가 있습니다   let과 const는  ES6에서 도입된 문법인데  let과 const 가 나온 이후로 var를 사용하는 것은  지양하고 있어요  이유는   https://www.youtube.com/watch?v=OCCpGh4ujb8&list=PLv2d7VI9OotTVOL4QmPfvJWPJvkmv6h-2&index=3 
이 영상에서 아주 유익하게 설명을 해주고 있어서  좀 길긴 하지만 한번 보시면 정리가 되실거에요 

변하는 값에는 let을  변치 않는 상수값에는 const를 사용합니다  여기서는 count_p는 아래 코드에서 값이 변하기에 
let을 사용하시면 됩니다

Reply:
이렇게 까지 자세하게 설명해 주시다니 감동입니다 ㅠㅠ